zk-sdk/src/sigma_proofs/grouped_ciphertext_validity/handles_2.rs

```diff
@@ -65,13 +65,14 @@ impl GroupedCiphertext2HandlesValidityProof {
     /// Note that the proof constructor does not take the actual Pedersen commitment or decryption
     /// handles as input; it only takes the associated Pedersen opening instead.
     ///
-    /// * `(destination_pubkey, auditor_pubkey)` - The ElGamal public keys associated with the decryption
-    /// handles
+    /// * `destination_pubkey` - The destination ElGamal public key
+    /// * `auditor` - The auditor ElGamal public key
```

Comment:
micro-nit: there's an inconsistency between spelling out `destination_pubkey` and `auditor` -- wanna change it to `auditor_pubkey`?
```suggestion
    /// * `destination_pubkey` - The destination ElGamal public key
    /// * `auditor_pubkey` - The auditor ElGamal public key
```

Reply:
Thanks for the catch!